README.md

```diff
@@ -1,27 +1,10 @@
 # ViewComponent::FragmentCaching
 With fragment caching in Rails, updates to a partial's source code will automatically bust
-appropriate caches in which the partial is detectable as a dependency. There are two types
-of dependencies.
+appropriate caches in which the partial is detectable as a dependency.
 
-### Render dependencies
-```ruby
-<%= render 'path/to/partial' %>
-<%= render Users::AuthorCardComponent.new(...) %>
-<%= render @user %>
-```
-In the second and third examples, ActionView will guess that the associated partial is at
-`/users/_user`.
-
-### Explicit dependencies
-```ruby
-<%# Template Dependency: 'path/to/partial' %>
-<%= render some_object %>
-```
-
-
-
-## Usage
-How to use my plugin.
+This gem augments ActionView's fragment caching strategy to detect and parse view components. In addition to digesting a
+component's view file (if present), it will also digest the component's ruby file and those of any superclasses
+descended from `ViewComponent::Base`. 
```

Comment:
I did like the note of how the second and third examples from the deleted section would resolve to a `users/_user` partial template. But I think this is good.